source/clear-linux/how-to/swupd/run.rst

```diff
@@ -1,59 +1,60 @@
 .. _swupd-run:
 
-How to run the updater
-######################
+How to update Clear Linux
+#########################
 
-This section describes how to manually trigger the software update code in
-Clear Linux\* OS for Intel® Architecture.
+This section describes how to update |CLOSIA|.
 
 Prerequisites
 =============
 
--  System is up and running a Clear Linux OS for Intel
-   Architecture official build, 7420 later.
+* The system is up and running the Clear Linux 15400 build or later.
```

Comment:
Should this document explain how to establish build (or should that be "release"?) the system is running at currently (`sudo swupd update -s`)?